styles.css

```diff
@@ -797,3 +797,20 @@ textarea {
 .width-90 {
   max-width: 90%;
 }
+
+.error-box {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: small;
+  text-align: justify;
+  line-height: 1.5rem;
+  color: white;
+  padding: 0.5rem;
```

Comment:
- [ ] [ **OPTIONAL** ] : Good job so far by implementing this error message alert. But, I think to improve the UX and good interaction with the user it would be better to change the text color to red. Red color makes it easier for users to see that there is an error.